### PR TITLE
Added editorconfig file

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.go]
+indent_style = tab
+indent_size = 8
+
+[*.tmpl]
```
Comment (thibaultmeyer) [inline .editorconfig:15]: Is these section necessaries ? I dont find any `tmpl` `less` `js` files on this repo

**Review by thibaultmeyer — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (tboerger) [reply .editorconfig:15]: I have taken exactly the same config we got for gitea

**Comment by andreynering** [comment]
LGTM

File: .editorconfig (hunk 1 above)
Comment (thibaultmeyer) [reply .editorconfig:15]: Could you remove unused sections ? This repos will never contains `tmpl` `less` or `js` files

Comment (metalmatze) [reply .editorconfig:15]: I don't really think they are cumbersome. In fact it makes it more easy to just copy paste `.editorconfig` from gitea into all other projects and create PRs.

**Comment by metalmatze** [comment]
LGTM

**Pull request merged**